src/components/Icon/icons/Attach.jsx

```diff
@@ -1,13 +1,9 @@
-/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
-/* tslint:disable:max-line-length */
-import * as React from 'react';
+import React from 'react';
```

Comment:
Wouldn't we need the copyright line in these files?

Reply:
yes we do, there's a TODO npm script to add the file headers after the files are generated. i hadn't found an existing module to do that easily yet